seed/lib/merging/merging.py

```diff
@@ -83,7 +92,90 @@ def get_state_attrs(state_list):
         return get_taxlotstate_attrs(state_list)
 
 
-def merge_extra_data(b1, b2, default=None):
+def _merge_relationships(merged_state, state1, state2):
+    """
+    Merge together the old relationships with the new.
+    """
+    # get some items off of this property view
+    no_measure_scenarios = [x for x in state2.scenarios.filter(measures__isnull=True)] + \
+        [x for x in state1.scenarios.filter(measures__isnull=True)]
+    building_files = [x for x in state2.building_files.all()] + [x for x in state1.building_files.all()]
+    simulations = [x for x in Simulation.objects.filter(property_state__in=[state1, state2])]
+    measures = [x for x in PropertyMeasure.objects.filter(property_state__in=[state1, state2])]
+
+    # TODO: dedup the relationships, if they are exact then don't add
```

Comment:
Is that just adding a line in each of the following loops to check some attributes for duplication?

Reply:
i actually implemented the deduping in the measures section below.